components/Tabs/Tabs.js

```diff
@@ -59,4 +59,5 @@ class TabItem {
 
 */
 
-links = document.querySelectorAll();
\ No newline at end of file
+const links = document.querySelectorAll(".tabs-link");
+links.forEach(link => new TabLink(link));
```

Comment:
glad the night study session gave you clarity, try and emulate these comments that were put in this repo